app/src/main/res/drawable/ic_menu_left.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_menu_left` appears to be unused
 Unused resources make applications larger and slow down builds.